Tools/scripts/extract_features.py

```diff
@@ -142,7 +143,7 @@ def __init__(self, filename, nm="arm-none-eabi-nm", strings="strings"):
 
             ('HAL_PARACHUTE_ENABLED', 'AP_Parachute::update',),
             ('AP_FENCE_ENABLED', r'AC_Fence::check\b',),
-            ('HAL_RALLY_ENABLED', r'AP_Rally::get_rally_max\b',),
+            ('HAL_RALLY_ENABLED', 'AP_Rally::find_nearest_rally_point',),
```

Comment:
I noticed Rally was not being detected by default? `get_rally_max` is not showing when running `arm-none-eabi-nm --demangle --print-size` - so I've moved this to a function that is visible and used.

Reply:
Compiler was probably inlining; you could try recompiling with a different optimisation level to see if that's true.  But it can't possibly be being elided given where it is used!